ledger/narwhal/transmission-id/src/bytes.rs

```diff
@@ -62,7 +59,6 @@ mod tests {
             // Check the byte representation.
             let expected_bytes = expected.to_bytes_le().unwrap();
             assert_eq!(expected, TransmissionID::read_le(&expected_bytes[..]).unwrap());
-            assert!(TransmissionID::<CurrentNetwork>::read_le(&expected_bytes[1..]).is_err());
```

Comment:
This will always be fixed size. You can keep it.

Reply:
It may be fixed size for each variant individually, but the variants are of different length and this test has been flaky in the past due to this assertion.